### DISCUSSION TITLE
存在多个VoidField，VoidField下嵌套内容，切换VoidField后，数据没有渲染

### DISCUSSION THREAD

**Discussion by AsortKeven** [alibaba/formily#3108] [Q&A 2.x] [answered]
复现步骤：点击”第二步“按钮  currentStep切换到2，第二步数据不渲染

复现链接：https://codesandbox.io/s/formilyce-shi-bevf9v?file=/App.tsx

**Comment by janryWang** [comment]
怎么复现的，没看懂复现步骤

**Reply by AsortKeven** [reply]
顶部有按钮“第一步”、“第二步”；
点击第二步，预期是currentStep切换到2，渲染第2步的数据；
实际结果是currentStep切换到了2，但是数据还是第一步的数据

**Comment by janryWang** [comment, marked as answer]
https://codesandbox.io/s/formilyce-shi-forked-2ribjy 写法问题，这是正确姿势
